nest/02.introduction.md

```diff
@@ -0,0 +1,57 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+A package is a compressed file archive containing all of the files coming with a particular software or application.
+
+A package manager allows the user to install, upgrade or remove software packages on their OS, in other words, to **manage the packages**.
+It is similar to an app store for any smartphone, or the Windows Store for example. One of the main differences between those and Nest (and most package managers), is that the upgrading must be manually commanded.
+On a smartphone, the applications are usually automatically upgraded overnight when you're charging your phone, if an update is available.
+Here, it's up to the user to upgrade their installed packages whenever they want or need to.
+
+To use Nest, the only interface available for now is the Command Line Interface (CLI).
+It might be a bit nerve racking for someone who has never seen a terminal or typed any command before, but don't worry, this guide is here to help you feel comfortable!
+
+## 1.2 Packages and dependencies
+Often, a package requires another one to work properly.
+Say, for example, someone develops a messenging application, which relies on encrypted messages to provide communication between two users.
+This application will probably use a package, written by someone else, to encrypt the messages sent with their app.
```

Comment:
This example is really complicated. Maybe something simpler a package using a programming language or a GUI toolkit would be clearer?

Reply:
I think using the discord app as an example could be fine, it has some flashy dependencies that seem obvious even for a non-technical user: a library to render web pages, another to send requests over the internet etc. 

(I don't think you need to be specific, being vague is enough)

Reply:
Yes, I'd just avoid the use of concepts that sound complicated (encryption).